test/shared/include/Rand.hpp

```diff
@@ -0,0 +1,318 @@
+#pragma once
```

Comment:
Are we including the Rand files from somewhere else?

Reply:
no